frameworks/cassandra/src/main/java/com/mesosphere/sdk/cassandra/scheduler/CassandraRecoveryPlanOverrider.java

```diff
@@ -81,6 +68,20 @@ private Phase getNodeRecoveryPhase(Plan inputPlan, int index) {
         TaskSpec taskSpec = podSpec.getTasks().stream().filter(t -> t.getName().equals("server")).findFirst().get();
         CommandSpec command = taskSpec.getCommand().get();
 
+        // Get IP address for the pre-existing node.
+
+        Optional<Protos.TaskStatus> status = StateStoreUtils.getTaskStatusFromProperty(
```

Comment:
upgrade compatibility check: will the TaskStatuses we get during reconciliation contain IPs?